PP_APP/server/frontend/templates/home.html

```diff
@@ -78,17 +79,11 @@ <h5 class="card-title">Some are not</h5>
       </div>
     </div>
 
-    <h2 class="align-items-center text-center">&nbsp;</h2>
-    <h2 class="align-items-center text-center">&nbsp;</h2>
-    <h2 class="align-items-center text-center">&nbsp;</h2>
-</div>
-
-<div class="container">
-  <div class="row py-lg-5">
-    <div class="col-md my-auto mx-auto">
-      <p style="font-size: 20px">Like us, many people are bothered by the poor packaging practices of companies.</p>
-    </div>
+  <h2 class="align-items-center text-center">&nbsp;</h2>
+  <h2 class="align-items-center text-center">Like us, many people are bothered by the poor packaging practices of companies.</h2>
```

Comment:
Is there a way to justify this paragraph so that it shows nicely no matter what the screen size? On my screen write now everything is written on one line except "comapnies".

Reply:
I suppose I can use REMs as a quick fix instead of pixels.